### PR TITLE
{data}[intel/2020b] Arrow v0.17.1 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3504.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/c8b3c810866acea1e69a7eb86fcdaf19 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2608.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/1f465ad601406646238211c21b379c54 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3134.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/d80ea04ba1c169034dba69cf2294f270 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node374.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/04d48efceb05b81fc633c627e1902887 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node100.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/682b6bf8fdc9bc978ba999f8ec18c124 for a full test report.

File: easybuild/easyconfigs/a/Arrow/Arrow-0.17.1-intel-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+easyblock = 'CMakeMake'
+
+name = 'Arrow'
+version = '0.17.1'
+
+homepage = 'https://arrow.apache.org'
+description = """Apache Arrow (incl. PyArrow Python bindings)), a cross-language development platform
```
Comment (lexming) [inline easybuild/easyconfigs/a/Arrow/Arrow-0.17.1-intel-2020b.eb:7]:
```suggestion
description = """Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform
```

Comment (lexming) [reply easybuild/easyconfigs/a/Arrow/Arrow-0.17.1-intel-2020b.eb:7]: @boegel I have to fix this typo, or I won't sleep at night

**Review by lexming — APPROVED** [review]
LGTM

File: easybuild/easyconfigs/a/Arrow/Arrow-0.17.1-intel-2020b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/a/Arrow/Arrow-0.17.1-intel-2020b.eb]: :+1:

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**